scripts/build/clean.js

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
I think this script needs a comment explaining its goal. In particular I'm confused why we're seemingly leaving `dist` intact? Your previous approach of `git clean -fX` would have removed that, right?